packages/itwinui-css/src/field.scss

```diff
@@ -28,6 +28,11 @@
   --_iui-field-state--disabled: var(--_iui-field-state,);
   --_iui-field-state--disabled-hover: var(--_iui-field-state,);
 
+  --_iui-button-icon-color: var(--_iui-field-state--default, var(--_iui-field-color-icon))
```

Comment:
I think `--_iui-button-icon-color` does not belong inside `field.scss`. This should probably be its own variable.